README.md

```diff
@@ -123,7 +126,6 @@ No deployment instructions yet.
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228913787-032509f9-e5c8-41bb-833c-ad1311e9b9ed.png)

- [ ] It would be nice if you add PostgreSQL in the prerequisites because this is the only thing which is needed to use this application 👍